ansible/playbooks/roles/grafana/defaults/versions.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Is this extra file really needed? I don't think there is any other place we actually have this and it only adds more code as we need to load this separately.

Reply:
This is needed because main.yml expects variables from elsewhere (probably a bad design).
I had discussion with @to-bar and we agreed on that.